docs/spec/spec.md

```diff
@@ -2017,7 +2028,7 @@ Following is the output of the server when a request is processed:
 ```
 
 #### 11.3.1 Service Level Interceptors
-The service level interceptors are applied to all the resolvers in the GraphQL service. The GraphQL module accept an array of service level interceptors, and it should be inserted as mentioned in the [Service Level Interceptor](#1016-service-level-interceptors) section.
+The service level interceptors are applied to all the resolvers in the GraphQL service. The GraphQL module accepts a single service level interceptor or an array of service level interceptors, and it should be inserted as mentioned in the [Service Level Interceptor](#1016-service-level-interceptors) section.
```

Comment:
We should add an example of providing multiple interceptors as well.

Reply:
Already there is following example

###### Example: Array of Service Level Interceptors

```ballerina
@graphql:ServiceConfig {
    interceptors: [new Interceptor1(), new Interceptor2()]
}
service on new graphql:Listener(9090) {
    // ...
}
```